mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_4790 net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder
-	FIELD field_22204 GRAVEL Lnet/minecraft/class_2680;
-	FIELD field_22209 SOUL_SAND Lnet/minecraft/class_2680;
+	FIELD field_22204 VALLEY_GRAVEL Lnet/minecraft/class_2680;
+	FIELD field_22209 VALLEY_SOUL_SAND Lnet/minecraft/class_2680;
```

Comment:
Why?

Reply:
`GRAVEL` and `SOUL_SAND` already exist in a superclass as final, thus these need unique names.

Reply:
Superclass final fields can be shadowed with the same name.